virttest/env_process.py

```diff
@@ -793,13 +793,13 @@ def preprocess(test, params, env):
     if kvm_userspace_ver_cmd:
         try:
             kvm_userspace_version = avocado_process.system_output(
-                kvm_userspace_ver_cmd, shell=True).strip()
+                kvm_userspace_ver_cmd, shell=True).decode().strip()
```

Comment:
I think it should be handled by `process.system_output()`, otherwise, each place which invoked this function would acquire this change.

Reply:
that's true,but process.system_output was ever changed by someone to return bytes forcedly. Not sure whether there is special consideration.
But anyway, I will submit one patch to restore return value to text